### ISSUE TITLE
Recycle bin

### ISSUE THREAD

**Issue by BruceZCQ** [wekan/wekan#1429]
Sometimes accidentally deleted the card, the hope can put them in the Sometimes accidentally deleted the card, the hope can put them in the bin, can restore bin, can restore. or remove the "delete"  the card / list function.

**Comment by xet7** [comment]
@BruceZCQ 

For that purpose, there is archive card and list, and restore:
https://github.com/wekan/wekan/wiki/Features#lists-add-archive-and-restore-archived-delete-list

Deleting really deletes permanently.

**Comment by BruceZCQ** [comment]
No, I mean because I accidentally deleted the card, can't recover, if can have a recycle bin, can restore a accidentally deleted.

**Comment by BruceZCQ** [comment]
remove follow code
`'click .js-delete': Popup.afterConfirm('cardDelete', function () {
    Popup.close();
    Cards.remove(this._id);
    Utils.goBoardId(this.boardId);
  }),`

or 
`'click .js-delete': Popup.afterConfirm('cardDelete', function () {
    Popup.close();
    Cards.putToTrash(this._id); //Cards.remove(this._id);
    Utils.goBoardId(this.boardId);
  }),`

**Comment by xet7** [comment]
@BruceZCQ 

Do you mean that "Archive this card" functionality is too complicated and hard to understand?

Should these be renamed?
- "Archive this card or list" => "Delete this card or list"
- "Archived cards/lists" => "Recycle Bin"

And should there be "Empty Recycle Bin" on top of those?

Then, if somebody accidentally empties Recycle Bin or some card/list there, is it OK ?
Or should there be additional recovery steps?

@ocdtrekkie 

What do you think?

**Comment by BruceZCQ** [comment]
@xet7 @ocdtrekkie 

I don't mean to close before ‘Archive’ function, also is not to say that they are not easy to understand, or need to rename the funcation. 

I just want a new function, somebody deleted cards or lists move to the recycle bin, the people manage their own recycle bin.

‘Archive’ or ‘Archived’ just store the archived cards or lists. if I delete the card or list , there is not exist in the Archived list. so I must to recreate the card or list agagin.

**Comment by xet7** [comment]
@hever 

For me, Recycle Bin is more familiar than Archive. I really like it.

Trello seems to have Archive button. I did not find a way to archive with drag and drop.

**Comment by hever** [comment]
@xet7 I'm sory, not Trello. Taiga.

**Comment by hever** [comment]
About Archive vs Recycle bin - I'm user of Gmail. There is Recycle bin place where you move email when you want to delete it. And you can move it to the Archive, when you want to store it forever. And have ability to find it in future. Maybe, it's something what i would like to have in wekan. Users may think that after some time cards in Recycle bin will be deleted automatically.

**Comment by xet7** [comment]
@hever 

In Windows, Recycle bin is not emptied automatically. It's more like that, Windows files and or Wekan cards. Throwing card to trash, restoring it back to board.

There is no "Move to archive" in Gmail and Windows.

**Comment by hever** [comment]
So maye it cloud be good, if there will be recycle bin and archive together. Archive for that you want to store forever and recycle bin for that you want to delete maybe after some time.

**Comment by ocdtrekkie** [comment]
I don't think Wekan should auto-perma-delete anything. Email auto-deletes from trash, but that makes sense given the volume of email people get. No operating system would delete your files automatically though, no matter how long they're in the trash.

Archive does lend itself to mean something you're likely to keep historically, and Recycle Bin/Trash suggests you are getting rid of it, but unless there's a difference in functionality, arguing about which to use is more or less bikeshedding.

**Comment by xet7** [comment]
You can rename some board like "Archive 2017" and move lists etc to it.

**Comment by xet7** [comment]
No, Recycle Bin will stay as Recycle Bin. This is now added to FAQ, with reasons why.
https://github.com/wekan/wekan/wiki/FAQ#policy-about-wekan-ui-changes

**Comment by hever** [comment]
Hey, Gmail's auto emptying recycle bin after 30 days is not bug. It's normal function and has sense.

But okay, cards in recycle bin doing slow down of board so it's not good for archive. I understand. So there's could be feature request for archive function that create new board (where members will be every time same as in original board) with one list archive and archiving cards to this place. Would it be?

**Comment by xet7** [comment]
@hever 

I added IFTTT Rules feature request #1996 . When card is moved to some list, it's only possible to automatically move it to Recycle Bin, not other board yet.

Well, when I can find option to stop Gmail from deleting emails automatically from Recycle Bin, I will stop calling it a bug. Windows Recycle Bin afaik does not automatically delete files. And I do not know any other software that automatically deletes something.

**Comment by hever** [comment]
We have recycle bin in office and the cleaner comes up from time to time and emptied it - and it's not bug it's valid. The same behavior in my home (there's no cleaner but wife do the job). Everywhere is recycle bin a place where you do not know day or hour if everything you put in is still there. Gmail has clearly rules and there is no sense to disable it. If you want it, you do something bad. If you want to review what you earlier don't want before you lost it forever then move it to some folder but no to recycle bin (also in home, office, etc.). Moving something to recycle bin means, you are going to risky way and you can lost it (wifes and cleaners unpredictable).

**Comment by xet7** [comment]
@hever 

Sorry Wekan still has only Recycle Bin.

I added Feature Request for unlimited cards and unlimited archive #1999
